### ISSUE TITLE
balancing factor

### ISSUE THREAD

**Issue by sjw821** [JulietLJY/BAL#2]
Thank you for sharing the code for an excellent paper. 
But, I am wondering how the optimal value of of 'Beta' is determined.
I can't find the code where the Beta is adaptively updated. It seems to be fixed at the initial value.

**Comment by JulietLJY** [comment]
Hi, thanks for your interest in our work. In our paper, we select the optimal ```beta``` by evaluating the results of the first epoch. We have updated the hyper-parameter ```beta``` outcomes which you can directly utilize. Please refer to [Readme.md](https://github.com/JulietLJY/BAL?tab=readme-ov-file#hyper-parameters) for details.

**Comment by sjw821** [comment]
Thank you for the reply. You mean that the best beta among several candidates is selected using the results of the first epoch. 
How are the results of the first epoch evaluated? Accuracy on the test set or another seperate validation set?
